src/components/detail/Detail.js

```diff
@@ -81,14 +81,16 @@ class Detail extends React.Component {
                 errorMessage={props.errorMessage}
               />}
             </div>
+            {props.isForgotPasswordSuccessful && <div className="alert alert-success" role="alert">{strings.RESET_PASSWORD_FOR_THIS_USER_SUCCESS}</div>}
```

Comment:
Line 84 and 85 seems duplicates
Maybe make a `const successMessage = resetPassSucces || updateSuccess || ''`
And then do `successMessage && div(successMessage)`